src/utils/CoinbasePixel.ts

```diff
@@ -213,13 +216,28 @@ export class CoinbasePixel {
         this.log('Received message: pixel_ready');
         this.isLoggedIn = !!data?.isLoggedIn as boolean;
 
+        this.removeErrorListener?.();
         this.sendAppParams(() => {
           this.onReadyCallback?.();
         });
       },
     });
   };
 
+  private addErrorListener = (): void => {
+    this.removeErrorListener = this.onMessage('error', {
```

Comment:
Do we need to have removeErrorListener both `unsub()` here & this.unsubs.remove(unsub)`?

Reply:
`removeErrorListener` will unsub this event listener when initialization is _successful_. I'm thinking that the consumer's callback should only ever be called once, and that should only happen during initialization, _before_ calling the `.open` method.

Reply:
That all sounds right to me, but I'm thinking about the fact that `this.unsubs` will still have en entry for this unsubscriber after we've called it. It's definitely a smaller thing - I should've prefaced this comment with `nit:`

Reply:
Yea. `this.unsubs` will end up calling `window.removeEventListener(event, unsubFxn)`. That should be harmless if the listener was already removed.